MAINTAINERS.md

```diff
@@ -26,7 +26,20 @@ After doing that commit with a message like "add foo-bar" and send a pull reques
 The continuous integration job will do some checks to see if your package's dependencies are up-to-date.
 
 The CI job notably doesn't compile packages, run tests, build documentation, or find missing C libraries.
-If you want to be proactive or if CI fails, you can make sure that your package builds against the newest versions of all dependencies like this:
+If you want to be proactive or if CI fails, you can make sure that your package builds against the latest nightly:
+
+```
+$ stack build --resolver nightly --haddock --test --bench --no-run-benchmarks
+```
+
+Or, if some dependencies haven't made it into Stackage Nightly yet:
+
+```
+$ rm -f stack.yaml && stack init --resolver nightly --solver
```

Comment:
`rm -f stack.yaml` might seem odd or annoying to those attempting to do this in their dev directory. Perhaps we should also explicitly recommend `stack unpack $yourPackage-$ver && cd $yourPackage-$ver` when performing these tests.

Reply:
Oh, that's brilliant, it will also address the pretty common issue of missing files for the test suite in the sdist tarball. I'll update.